config/config.go

```diff
@@ -25,11 +25,24 @@ var (
 	pathConfig       = cue.MakePath(cue.Str("config"))
 )
 
-// Load loads the configuration at the given directory and returns it.
+type options struct {
+	tags []string
+}
+
+type optionFunc func(o *options)
+
+// WithTags returns an option function to provide a list of tag values when loading the configuration.
+func WithTags(tags []string) optionFunc {
+	return func(o *options) {
+		o.tags = tags
+	}
+}
+
+// Load loads the configuration at the given directory, using the specified options, and returns it.
 // On error, the error value may contain an Error value
 // containing multiple errors.
-func Load(configDir string) (*Config, error) {
-	cfg, err := load0(configDir)
+func Load(configDir string, opts ...optionFunc) (*Config, error) {
```

Comment:
... or, given that there's only one possible option so far, it has a nice zero value, and we're not concerned with maintaining future backward compatibility (at least I don't think we are!) just pass in `tags` directly?

Reply:
Yes, I agree – I hadn't fleshed it out fully yet, as I was still experimenting and started bottom up adding the changes, and this approach won't break existing code (at compilation).

Reply:
Done